src/h5web/providers/context.ts

```diff
@@ -11,4 +11,7 @@ export const ProviderContext = createContext<{
   domain: string;
   metadata: HDF5Metadata;
   getValue: ProviderAPI['getValue'];
+  getValues: (
+    ids: (HDF5Id | undefined)[]
+  ) => Promise<(HDF5Value | undefined)[]>;
```

Comment:
`getValues` is then not consistent with `getValue` that does not accept `undefined` id but I found no other alternatives: https://github.com/silx-kit/h5web/pull/256/files#r509287208